packages/to-json-schema/src/convertAction.ts

```diff
@@ -172,6 +219,21 @@ export function convertAction(
       break;
     }
 
+    case 'starts_with': {
```

Comment:
You can check this from here - https://json-schema.org/understanding-json-schema/reference/string#regexp

Reply:
Thank you! We need to check if special string characters can cause bugs, and if there is an easy way to detect and escape them if necessary.

Reply:
Shall I add some tests for this, then?

Reply:
Let me do some tests fist. I get back to you later.

Reply:
I think we can't support this feature until [`Regex.escape`](https://github.com/tc39/proposal-regex-escaping) is implemented. We could support it by adding our [own escape logic](https://github.com/lionel-rowe/regexp-escape-polyfill/blob/main/escape.mjs) but this would increase the bundle size of `toJsonSchema` from 4.56 kB (1.52 kB gzip) to 5.34 kB (1.93 kB gzip).

Reply:
Once this is resolved, I will do a final review and merge the PR.